src/registry/children/Children.scss

```diff
@@ -0,0 +1,13 @@
+@use '@utrecht/components/table';
+@use '@utrecht/components/custom-checkbox';
+
+@use '@/scss/bem';
+
+.openforms-children {
+  @include bem.modifier('with-manual-creating') {
```

Comment:
do we need this modifier at all?

If there are no errors, the node will not be rendered and thus not take up vertical space, no?

Reply:
and why can't we apply a grid/flexbox row gap for the children component? :thinking: